content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -8,15 +8,19 @@ menu:
     parent: corda5-api
     weight: 5000
 section_menu: corda5
-draft: true
 ---
 # net.corda.v5.crypto
 
-The `corda-crypto` module 
+The `crypto` package provides services and types for performing cryptographic operations. The following services are available:
```

Comment:
Only just noticed this, the `crypto` package only contains types and not services.

Perhaps we could say "types used by services in `net.corda.v5.application.crypto`" as a first link between the two.